File: exchange-desktop-hmekhatib-main/src/main/java/com/hmekhatib/exchange/api/model/Graph.java
package com.hmekhatib.exchange.api.model;

import com.google.gson.annotations.SerializedName;
import javafx.scene.chart.NumberAxis;

import java.util.List;

public class Graph {

        @SerializedName("x")
        public List<String> xaxis;
        @SerializedName("buy")
        public List<Float> yAxisB;
        @SerializedName("sell")
        public List<Float> yAxisS;


}

File: exchange-android/exchange-android-hmekhatib/app/src/main/java/com/hmekhatib/currencyexchange/api/model/Graph.kt
package com.hmekhatib.currencyexchange.api.model

import com.google.gson.annotations.SerializedName
import java.util.*

class Graph{
    @SerializedName("x")
    var xaxis: List<String>? = null
    @SerializedName("buy")
    var yaxisB: List<Float>? = null
    @SerializedName("sell")
    var yaxisS: List<Float>? = null

}